pkg/validation/config.go

```diff
@@ -460,8 +460,9 @@ func validateResourceList(fieldRoot string, list api.ResourceList) []error {
 	var numInvalid int
 	for key := range list {
 		switch key {
-		case "cpu", "memory":
-			if quantity, err := resource.ParseQuantity(list[key]); err != nil {
+		case "cpu", "memory", api.ShmResource:
```

Comment:
Does it makes sense to allow `ShmResource` in `limits`? Maybe it would make sense to validate it is only present in `requests` (or at least that is how I understand our intent).

Reply:
There is a k8s validation that makes us set the `Limit` as well on these custom resources. I wasn't aware of that when I wrote the doc up.

`could not run steps: step shm-increase failed: "shm-increase" test steps failed: failed to create or restart shm-increase-step-with-increased-shm pod: unable to create pod: Pod "shm-increase-step-with-increased-shm" is invalid: spec.containers[0].resources.limits: Required value: Limit must be set for non overcommitable resources`

In practice we should just have the config list the `Request` and the `Limit` the same.